fronts-client/src/components/FrontsEdit/FrontContent.tsx

```diff
@@ -143,6 +164,15 @@ class FrontContent extends React.Component<FrontProps, FrontState> {
 
   public render() {
     const { front } = this.props;
+
+    const isCollectionsStale =
+      this.props.collectionsLastFetch &&
+      Date.now() - this.props.collectionsLastFetch > TWENTY_SECONDS_IN_MILLIS;
```

Comment:
Whilst I'm not sure this is actually necessary in that we can check the error directly, I doubt it actually works as desired, as the only time its evaluated is when some thing causes it to re-render (which is likely fresh changes - at which point the lastFetch will be v. recent)... instead perhaps we use a `setTimeout` (with 20 second delay) to write some new field to state (something like `lastStaleCheck`) to force this to be re-evaluated (there might be a nice React way of doing that)... thoughts welcome....

Reply:
DONE in https://github.com/guardian/facia-tool/pull/1277/commits/da957b710fb4802b478d304481817b5acac588d4

Reply:
Then DONE better in https://github.com/guardian/facia-tool/pull/1277/commits/3e0fa48b3f54b82e4f4a4d082cab76d799ccac1b